nil pointer in openSRTP

### Your environment.
  - Version: 8ef5b7034ff4664530938fa48873d34d3f9672fc

### What did you do?
Created a PeerConnection between two Pion instances and sent data using `track.WriteSample`.

### What happened?

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x42ed3dd]

goroutine 32 [running]:
github.com/pions/webrtc.(*PeerConnection).openSRTP.func1(0xc000132480, 0xc0000004d2, 0x2)
        /Users/max/go/pkg/mod/github.com/pions/webrtc@v1.2.1-0.20190328075210-8ef5b7034ff4/peerconnection.go:991 +0x2bd
created by github.com/pions/webrtc.(*PeerConnection).openSRTP
        /Users/max/go/pkg/mod/github.com/pions/webrtc@v1.2.1-0.20190328075210-8ef5b7034ff4/peerconnection.go:962 +0x3fe
```

This is the line
```
			sdpCodec, err := pc.currentLocalDescription.parsed.GetCodecForPayloadType(receiver.Track().PayloadType())
			if err != nil {
				pc.log.Warnf("no codec could be found in RemoteDescription for payloadType %d", receiver.Track().PayloadType())
				return
			}
```

--- a/peerconnection.go
+++ b/peerconnection.go
@@ -546,6 +546,8 @@ func (pc *PeerConnection) CreateAnswer(options *AnswerOptions) (SessionDescripti
 	switch {
 	case options != nil:
 		return SessionDescription{}, fmt.Errorf("TODO handle options")
+	case pc.RemoteDescription() == nil:
+		return SessionDescription{}, &rtcerr.InvalidStateError{Err: ErrNoRemoteDescription}
 	case useIdentity:
 		return SessionDescription{}, fmt.Errorf("TODO handle identity provider")
 	case pc.isClosed:

--- a/peerconnection_go_test.go
+++ b/peerconnection_go_test.go
@@ -364,3 +364,14 @@ func TestPeerConnection_PeropertyGetters(t *testing.T) {
 	assert.Equal(t, pc.iceConnectionState, pc.ICEConnectionState(), "should match")
 	assert.Equal(t, pc.connectionState, pc.ConnectionState(), "should match")
 }
+
+func TestPeerConnection_AnswerWithoutOffer(t *testing.T) {
+	pc, err := NewPeerConnection(Configuration{})
+	if err != nil {
+		t.Errorf("New PeerConnection: got error: %v", err)
+	}
+	_, err = pc.CreateAnswer(nil)
+	if !reflect.DeepEqual(&rtcerr.InvalidStateError{Err: ErrNoRemoteDescription}, err) {
+		t.Errorf("CreateAnswer without RemoteDescription: got error: %v", err)
+	}
+}

